e2e/app/page/google-login.ts

```diff
@@ -79,14 +81,24 @@ export default class GoogleLoginPage {
   async enterPassword(pwd: string) : Promise<void> {
     const input = await this.password();
     await input.focus();
-    await input.type(pwd);
+    await input.type(pwd, {delay: 15});
+    await input.dispose();
   }
 
   /**
    * Click Next button to submit login credential.
    */
   async submit() : Promise<void> {
-    return (new Button(this.page, FieldSelector.SubmitButton)).clickAndWait();
+    const submitButton = new Button(this.page, FieldSelector.SubmitButton);
+    const [response] = await Promise.all([
+      this.page.waitForNavigation({waitUntil: ['networkidle2', 'load'], timeout: 180000}),
```

Comment:
what does 'networkidle2' mean?

Reply:
Both `networkidle0` and `networkidle2` are ways to determine page load state by checking number of open requests.
I am experimenting with `networkidle2` because sometimes tests fail in Home page after login. But nothing (checking screenshot and html file) wrong can be found. I suspect `networkidle0` has bug.